src/main/java/specifications/listSpecification/CreateListSpec.java

```diff
@@ -0,0 +1,16 @@
+package specifications.listSpecification;
+
+import io.restassured.specification.RequestSpecification;
+import specifications.BaseSpec;
+
+public class CreateListSpec extends BaseSpec {
+
+    public RequestSpecification getListCreateSpec(String name) {
+        return baseRequestBuilder
+                .setBasePath("/1/boards/{id}/lists")
+                .addQueryParam(parameterListName, name)
+                .setBody("")
```

Comment:
you can remove setBody because it is empty

Reply:
I can't because the test is failing

Reply:
then its a great place to use entity serialisation